.github/workflows/release.yml

```diff
@@ -169,7 +167,6 @@ jobs:
           GITHUB_REPO_SLUG: ${{ github.repository }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GITHUB_CONTEXT: ${{ github.event.pull_request.commits_url }}
-          PULP_LOGGING: docker
```

Comment:
@mdellweg do we need it on plugin-template?

Reply:
no.